components/content/simple-editor/SceneNodeBox.vue

```diff
@@ -0,0 +1,36 @@
+<script setup lang="ts">
+import { type Mesh, Color, MathUtils } from 'three'
+import type { BoxSceneNode } from './types'
+
+const props = defineProps({
+  properties: { 
+    type: Object as PropType<BoxSceneNode['properties']>, 
+    default: () => ({ width: 10, height: 10, length: 10 }), 
+  },
+  first: { type: Boolean, default: false },
+})
+
+// weird bug happening if we don't explicitly emit click instead of letting it flow through
```

Comment:
Bug to report?

Reply:
will try to create a repro and create an issue